java-chassis-distribution/src/release/LICENSE

```diff
@@ -461,3 +461,4 @@ Zipkin Sender: OkHttp 3 (https://github.com/openzipkin/zipkin-reporter-java/zipk
 Zipkin v1 (https://github.com/openzipkin/zipkin/) io.zipkin.java:zipkin:jar:2.4.2
 Zipkin v2 (https://github.com/openzipkin/zipkin/) io.zipkin.zipkin2:zipkin:jar:2.4.2
 zuul-core (https://github.com/Netflix/zuul) com.netflix.zuul:zuul-core:jar:1.3.0
+protobuf-java (https://github.com/protocolbuffers/protobuf/tree/master/java) com.google.protobuf:protobuf-java:jar:3.6.1
```

Comment:
This project does not using Apache License v2

Reply:
it's 3-Clause BSD license
fixed